MVVM/ViewModel/SafeMode_UIOverlay.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Godot;
+using RoverControlApp.Core;
+using RoverControlApp.MVVM.Model;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace RoverControlApp.MVVM.ViewModel;
+
+public partial class SafeMode_UIOverlay : UIOverlay
+{
+	[Export]
+	PanelContainer _panelContainer;
+
+	private int _inputMode;
+
+	private static float _speedLimit => LocalSettings.Singleton.SpeedLimiter.MaxSpeed;
+
+	public override Dictionary<int, Setting> Presets { get; } = new() 
+	{
+		{ 0, new(Colors.Blue, Colors.LightBlue, $"SpeedLimiter: ON {_speedLimit:P0}", "SpeedLimiter: ") },
+		{ 1, new(Colors.DarkRed, Colors.Orange, "SpeedLimiter: OFF", "SpeedLimiter: ") }
+
+	};
+
+	public Task ControlModeChangedSubscriber(MqttClasses.ControlMode newMode)
+	{
+		_inputMode = (int)newMode;
+		CallDeferred(MethodName.UpdateSafeModeIndicatator);
```

Comment:
Misspell
```suggestion
		CallDeferred(MethodName.UpdateSafeModeIndicator);
```

Reply:
Adressed in [163c4f2](https://github.com/wisniax/RoverControlApp/pull/36/commits/163c4f29a5609d21619e3936b263bbd5cac3fc49)